packages/fetchai/connections/p2p_libp2p/dht/dhtpeer/dhtpeer.go

```diff
@@ -526,8 +526,8 @@ func (dhtPeer *DHTPeer) RouteEnvelope(envel *aea.Envelope) error {
 }
 
 func (dhtPeer *DHTPeer) lookupAddressDHT(address string) (peer.ID, error) {
-	lerror, _, linfo, _ := dhtPeer.getLoggers()
-
+	lerror, lwarn, linfo, _ := dhtPeer.getLoggers()
+	var err error
 	addressCID, err := utils.ComputeCID(address)
```

Comment:
err will be overwritten in 531, intended?

Reply:
yes, it is